src/fury.es6

```diff
@@ -7,16 +8,17 @@ import legacyAPI from './legacy/blueprint';
 import legacyBlueprintParser from './legacy/blueprint-parser';
 import legacyMarkdownRenderer from './legacy/markdown';
 
-// Register Parse Result and API description primitives
-import './refract/parseResult';
+const minim = minimModule.namespace()
+  .use(minimParseResult);
```

Comment:
Just a comment, nothing to do here. I don't like the names `minim*` here. Wonder if there is a better way.